akka-persistence/src/main/scala/akka/persistence/PersistentView.scala

```diff
@@ -174,21 +178,26 @@ trait PersistentView extends Actor with Snapshotter with Stash with StashFactory
   private def updateLastSequenceNr(persistent: PersistentRepr): Unit =
     if (persistent.sequenceNr > _lastSequenceNr) _lastSequenceNr = persistent.sequenceNr
 
+  override def recovery = Recover(replayMax = autoUpdateReplayMax)
+
   /**
    * Triggers an initial recovery, starting form a snapshot, if any, and replaying at most `autoUpdateReplayMax`
    * messages (following that snapshot).
    */
   override def preStart(): Unit = {
-    super.preStart()
-    self ! Recover(replayMax = autoUpdateReplayMax)
-    if (autoUpdate) schedule = Some(context.system.scheduler.schedule(autoUpdateInterval, autoUpdateInterval,
-      self, ScheduledUpdate(autoUpdateReplayMax)))
+    startRecovery(recovery)
```

Comment:
TODO I was trying to rip it out into `aroundPreStart` without success, something weird is going on with the around calls...